config/common/events-dispatcher.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Psr\EventDispatcher\ListenerProviderInterface;
+use Yiisoft\EventDispatcher\Dispatcher\Dispatcher;
+use Yiisoft\EventDispatcher\Provider\Provider;
+
+return [
+    EventDispatcherInterface::class => Dispatcher::class,
```

Comment:
It is already specified in the package itself: https://github.com/yiisoft/event-dispatcher/pull/55/files#diff-2627613725287eb3e91715fe33517383caf76e4a25e5eb18dd9fc23b5dca9373R11. What's the goal of having it again here?